### DISCUSSION TITLE
GraphQL Query Error for previously available default branch of NOT_FOUND repo

### DISCUSSION THREAD

**Discussion by Nosferican** [community/community#24783] [Apps, API and Webhooks]
<p>The following repository is NOT_FOUND (deleted or made private) but resolves to the repository still. When accessing the ID of the previously available default branch it errors out rather than giving the NOT_FOUND error.</p>
<pre><code class="lang-auto">query Commits {
  node(id: "MDEwOlJlcG9zaXRvcnkyNDY1NzYyNDA=") {
    id
    ... on Repository {
      defaultBranchRef {
        id
      }
    }
  }
}
</code></pre>
<p>results in</p>
<pre><code class="lang-auto">{
  "errors": [
    {
      "message": "Something went wrong while executing your query. Please include `8F9A:7E02:4F61BE:FB19FE:6040640D` when reporting this issue."
    }
  ]
}
</code></pre>

**Comment by nethgato** [comment, marked as answer]
<p>Hey <a class="mention" href="/nosferican">@nosferican</a> :wave:</p>
<p>Is this still ongoing? I tried rerunning your query, and I get:</p>
<pre><code class="lang-auto">"message": "Could not resolve to a node with the global id of 'MDEwOlJlcG9zaXRvcnkyNDY1NzYyNDA='."
</code></pre>
<p>…and kicks out the <code>NOT_FOUND</code> error.</p>
<p>If you are still getting that error message which fails to run the query at all (so it seems) let us know!</p>

**Comment by Nosferican** [comment]
<p>Seems to have been fixed now. Thanks!</p>
